SYCL/ESIMD/api/saturation_smoke_aspect-fp64.cpp

```diff
@@ -0,0 +1,23 @@
+//==---- saturation_smoke_aspect-fp64.cpp  - DPC++ ESIMD on-device test
+//-----==//
```

Comment:
This looks wrong.

Reply:
This change is still required by the clang-format-check patch.

Reply:
Shorten the "---" parts and it will fit one line.